project/demos/arrive/ArriveDemo.gd

```diff
@@ -0,0 +1,37 @@
+extends Node2D
+
+
+onready var _target: = $Target
```

Comment:
I was just responding to a comment in a godot issue. We shouldn't use pseudo-private vars with `onready` since we mostly use `onready` for node dependencies and so anyone can do `child_node.get_node("Target")` anyway so we might as well let them do `child_node.target`